### PR TITLE
Profile create and edit frontend

### PR DESCRIPTION
**Pull request by OmegaOoh**
### Description

- UI for profile creation
- UI for profile pages
- Profile owners can edit profile from profile page
- profile can be get from backend using username instead of user id

### Functional Acceptance Criteria

- User get redirects to profile creation page on first login
- User get redirects back to where they are after they submit the profile
- User can edit their profile via profile page

### Checklist

- [x] I have performed a self-review of my code.
- [x] I have added tests to cover my changes.
- [ ] I have updated the documentation, if applicable.
- [x] My code follows the Coding Guidelines.

### Type of Change

- [x] Bug fix
- [x] New feature
- [ ] Documentation update
- [ ] Other (please specify)

### CODE REVIEW

File: frontend/src/functions/Authentications.js [hunk 1]
```diff
@@ -2,14 +2,17 @@ import { ref } from 'vue';
 import apiClient from '@/api';
 import { googleTokenLogin } from 'vue3-google-login';
 import { createPostRequest } from './HttpRequest';
-import { getCookie, setCookie, deleteCookie } from './cookies';
+import { getCookie, setCookie, deleteCookie } from './Cookies.js';
```
Comment (opxz7148) [inline frontend/src/functions/Authentications.js:5]:
```suggestion
import { getCookie, setCookie, deleteCookie } from './Cookies.js';
```
I'm not sure what happen here but './Cookies.js'; with capital C cause a Module not found error in my machine.

**Review by opxz7148 — CHANGES_REQUESTED** [review]
Over all profile page UI look really nice but there are minor problem that I've found as I stated in comment

File: frontend/src/functions/Authentications.js (hunk 1 above)
Comment (OmegaOoh) [reply frontend/src/functions/Authentications.js:5]: I think it is something weird about git, I already changed its name to Cookie to follow other files. but Seems like it doesn't updates on git

**Review by opxz7148 — APPROVED** [review]
Now it can run properly. Nice work _Approve_

**Review by TeerapatTrepopsakulsin — COMMENTED** [review]
Extremely fast work. Impressive. I have some suggestions.

Suggestion
- The bio doesn't show full bio (Cut some text, doesn't show full 256 chars)
- Some profile information is a little bit weird, or missing (nickname, pronoun, username) (I suppose it's for next iter)
- The tab for showing joined activity can contain too much activity, and push `Create Activity` too far down, I think `Create Activity` should be above the joined activities

**Comment by OmegaOoh** [comment]
> Extremely fast work. Impressive. I have some suggestions.
> 
> Suggestion
> 
> * The bio doesn't show full bio (Cut some text, doesn't show full 256 chars)
> * Some profile information is a little bit weird, or missing (nickname, pronoun, username) (I suppose it's for next iter)
> * The tab for showing joined activity can contain too much activity, and push `Create Activity` too far down, I think `Create Activity` should be above the joined activities

The drawer only contains 5 recent activity

**Review by Thanawas-Sirilertsathit — APPROVED** [review]
Approve. Everything works fine and chat fixed to be able to send image without message text.

**Review by TeerapatTrepopsakulsin — APPROVED** [review]
Nicely done! _Approve_

**Pull request merged**